scripts/core_scripts.lua

```diff
@@ -137,3 +165,150 @@ openspace.invertBooleanProperty = function(propertyIdentifier)
     end
     openspace.setPropertyValueSingle(propertyIdentifier, not value)
 end
+
+openspace.fadeIn = function(identifier, fadeTime, endScript)
+    -- Set default values for optional arguments
+    if endScript == nil then
```

Comment:
Lua-esque way of writing this:
`endScript = endScript or ""`

same for `fadeTime`